tools/mmp/driver.cs

```diff
@@ -408,9 +408,20 @@ static void Main2 (string [] args)
 			if (!targetFramework.HasValue)
 				targetFramework = TargetFramework.Default;
 
-			if (TargetFramework.Identifier == TargetFramework.Xamarin_Mac_2_0.Identifier) {
+			if (TargetFramework == TargetFramework.Xamarin_Mac_2_0_Mobile || TargetFramework.Version == TargetFramework.Xamarin_Mac_2_0_Mobile.Version) {
```

Comment:
No. It was just hard coded in an invocation in a makefile.

However, it seemed sane enough that I figured I should just honor it.